cms/djangoapps/contentstore/tasks.py

```diff
@@ -52,8 +52,8 @@ def rerun_course(source_course_key_string, destination_course_key_string, user_i
         # call edxval to attach videos to the rerun
         copy_course_videos(source_course_key, destination_course_key)
 
-        # initialize course settings like tags stc
-        initialize_course_settings(source_course_key, destination_course_key)
+        # Apply post re run creation tasks
```

Comment:
Let's, please remove this comment or make is more descriptive as it's just function's name.